app/forms/depositor_form.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+# Form for collection depositors
+class DepositorForm < CollectionParticipantForm
```

Comment:
Note that the nested attribute components require a form that matches the field name, so the parent `CollectionContributorForm` is used to avoid duplication in `DepositorForm` and `ManagerForm`. This will likely also be used in a future `ReviewerForm`.

Reply:
I'd be inclined to duplicate since DepositorForm isn't logically a child of CollectionParticipantForm.